### PR TITLE
Add feature for CSS Typed OM

### PR DESCRIPTION
**Pull request by queengooborg**
This PR adds a feature for the CSS Typed OM Level 1.

### CODE REVIEW

**Comment by jamesnw** [comment]
I'd propose a different approach- many of these keys should be part of separate features. For instance, `api.CSS.cap_static` should be part of the `cap` feature, similar to how the `highlight` feature contains `api.CSS.highlights_static`. 

This feature should contain the keys that are related to CSS syntax and wouldn't be part of other features. Some examples-

  - api.CSSNumericArray
  - api.CSSNumericValue
  - api.CSSUnitValue.value

Some of the keys here can be added to existing features, but there are others like `rcap` that will still need to be created.

**Comment by ddbeck** [comment]
> This feature should contain the keys that are related to CSS syntax and wouldn't be part of other features.

This is the core thing. I think a lot of the keys here could survive as a typed OM feature no matter what.

The feature-specific keys, those are the troublesome ones. I'm going to make a pragmatic suggestion: put the per-feature keys into a draft feature—call it the typed OM "junk drawer" if we must. Sort the keys by which "real" feature they ought to live with and comment the lines accordingly.

I think this is going to be one of the prime features for resolving https://github.com/web-platform-dx/web-features/issues/971. We'll need some way to declare inferior or interior or adjacent features and this seems like a likely candidate for taking it on, since we'll get a bunch of them at once. If we can deal with that soon, then great. If we get impatient, then we'll have done the work to decide where they'll need to be moved already.

**Comment by jamesnw** [comment]
Another option would be to pull out a `CSS numeric factory functions` feature, which would encompass all of items that would be part of another feature. This would essentially be the draft junk drawer idea, and we could add comments saying that, but it would also be a useful feature. See #2000 for a draft PR.

File: features/css-typed-om.yml [hunk 1]
```diff
@@ -0,0 +1,138 @@
+name: CSS typed object model
+description: The CSS typed object model allows web developers to programmatically interact with CSS values.
```
[the remark this answers is not in the document]
Comment (jamesnw) [reply features/css-typed-om.yml]: Thanks, I was mid-suggestion on a rewrite, but yours is superior.

**Comment by jamesnw** [comment]
@queengooborg I shepherded this over the finish line since I had made some parallel changes in the interim- thanks for the initial feature and patience!

**Pull request merged**